nncf/quantization/algorithms/channel_alignment/algorithm.py

```diff
@@ -97,6 +97,10 @@ def apply(
         statistic_points: Optional[StatisticPointsContainer] = None,
         dataset: Optional[Dataset] = None,
     ) -> TModel:
+        backend = get_backend(model)
+        if backend != BackendType.OPENVINO:
+            return model
```

Comment:
Should we add a warning as it was before?

Reply:
Done